### PR TITLE
[Digital Ocean] Add ability to speicify the private SSH key path

### PR DESCRIPTION
**Pull request by v4n**
When specifying a ssh key fingerprint the driver defaults to using `~/.ssh/id_rsa`. 
This change will allows the user to specify where the private key is located .

### CODE REVIEW

**Comment by v4n** [comment]
@nathanleclaire we could probably also generate a public key from the `digitalocean-ssh-key-path` and upload that to DO when no `digitalocean-ssh-key-fingerprint` is provided.

For now I wanted to keep things simple. IMO it's essential to be able to specify the ssh key path when specifying the fingerprint. It is common for people to have multiple private keys in their ssh folder.

**Comment by v4n** [comment]
@nathanleclaire is it worth it fixing the branch conflicts? Do you recommend another approach? Or think this is not necessary and we should close this?

**Comment by nathanleclaire** [comment]
It's OK with me to have this feature and it's similar to what other drivers have IIRC.

If you will rebase and please submit docs change to https://github.com/docker/docker.github.io (`vnext-machine` branch) instead of in this PR, I would be happy to merge it.

**Comment by nathanleclaire** [comment]
Sorry on the late code review feedback.  I thought I posted it a while ago but GH new review feature took some getting used to.

File: docs/drivers/digital-ocean.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Options:
 -   `--digitalocean-ssh-user`: SSH username.
 -   `--digitalocean-ssh-port`: SSH port.
 -   `--digitalocean-ssh-key-fingerprint`: Use an existing SSH key instead of creating a new one, see [SSH keys](https://developers.digitalocean.com/documentation/v2/#ssh-keys).
+-   `--digitalocean-ssh-key-path`: Local path of the Digital Ocean public key. `--digitalocean-ssh-key-fingerprint` must be provided with this option.
```
Comment (v4n) [inline docs/drivers/digital-ocean.md]: @nathanleclaire meant the private key. Will fix this on the commit to `docker/docker.github.io`.

**Comment by v4n** [comment]
@nathanleclaire no problem. Will be great to stop using the custom builds once this gets out.

I've fixed the rebase issue and will do a commit to `docker/docker.github.io/tree/vnext-machine` in a bit.

**Comment by nathanleclaire** [comment]
LGTM

**Comment by johndmulhausen** [comment]
@v4n @nathanleclaire @dubstrike What version of Docker Machine are these docs changes for and when, approximately, is it shipping? Would like to track with a milestone. :)

**Comment by nathanleclaire** [comment]
0.9.0 alongside Docker 1.13.0.  The Machine releases are always released in tandem with the Docker Engine.

**Pull request merged**